base-models/src/main/java/com/telecominfraproject/wlan/core/model/utils/DecibelUtils.java

```diff
@@ -19,6 +21,13 @@ public static double getAverageDecibel(int[] decibels)
         return avgDbVal10;
     }
 
+    public static double getAverageDecibel(Collection<Integer> decibels) 
```

Comment:
this is readable, I give you that, but it makes an unnecessary copy of the array.
How about something like this:
`
    public static double getAverageDecibel(Collection<Integer> decibels)
    {
        
        if (decibels == null || decibels.isEmpty()) {
            return 0.0;
        }
        
        double aggregation = 0;
        int count=0;
        Iterator<Integer> iter = decibels.iterator();
        
        while(iter.hasNext())
        {
            Integer value = iter.next();
            if(value!=null) {
                aggregation += Math.pow(10, value/10.0);
                count++;
            }
        } 

        double avgDbVal10 = 10 * Math.log10(aggregation/count);
        return avgDbVal10;
    }
`